MaxText/configs/base.yml

```diff
@@ -63,11 +63,13 @@ logical_axis_rules: [
                     ]
 data_sharding: [['data', 'fsdp', 'tensor']]
 
-dcn_data_parallelism: -1
+# One axis for each parallelism type may hold a placeholder (-1)
```

Comment:
Can we mention that the default is to set the product of DCN axis (e.g. dcn_data_parallelism) to the number of slices and the ICI axis (e.g. ici_fsdp_parallelism) to the number of devices per slice?